x-pack/plugins/aiops/public/components/change_point_detection/fields_config.tsx

```diff
@@ -138,25 +158,244 @@ export interface FieldPanelProps {
+              defaultMessage: 'Attach selected charts',
+            })
+          : i18n.translate('xpack.aiops.changePointDetection.attachChartsLabel', {
+              defaultMessage: 'Attach charts',
+            }),
+      items: [
+        {
+          name: i18n.translate('xpack.aiops.changePointDetection.attachToDashboardLabel', {
+            defaultMessage: 'To dashboard',
+          }),
+          panel: 'attachToDashboardPanel',
+        },
+        {
+          name: i18n.translate('xpack.aiops.changePointDetection.attachToCaseLabel', {
+            defaultMessage: 'To case',
+          }),
+          disabled: caseAttachmentButtonDisabled,
+          ...(caseAttachmentButtonDisabled
+            ? {
+                toolTipPosition: 'left',
+                toolTipContent: i18n.translate(
+                  'xpack.aiops.changePointDetection.attachToCaseTooltipContent',
+                  {
+                    defaultMessage: 'Select change points to attach',
+                  }
+                ),
+              }
+            : {}),
+          onClick: () => {
+            openCasesModalCallback({
+              timeRange,
+              fn: fieldConfig.fn,
+              metricField: fieldConfig.metricField,
+              dataViewId: dataView.id,
+              ...(fieldConfig.splitField
+                ? {
+                    splitField: fieldConfig.splitField,
+                    partitions: selectedPartitions,
+                  }
+                : {}),
+            });
+          },
+        },
+      ],
+    },
+    {
+      id: 'attachToDashboardPanel',
+      title: i18n.translate('xpack.aiops.changePointDetection.attachToDashboardTitle', {
+        defaultMessage: 'Attach to dashboard',
+      }),
+      size: 's',
+      content: (
+        <EuiPanel paddingSize={'s'}>
+          <EuiSpacer size={'s'} />
+          <EuiForm>
+            <EuiFormRow fullWidth>
+              <EuiSwitch
+                label={i18n.translate('xpack.aiops.changePointDetection.applyTimeRangeLabel', {
+                  defaultMessage: 'Apply time range',
+                })}
+                checked={dashboardAttachment.applyTimeRange}
+                onChange={(e) =>
+                  setDashboardAttachment((prevState) => {
+                    return {
+                      ...prevState,
+                      applyTimeRange: e.target.checked,
+                    };
+                  })
+                }
+                compressed
+              />
+            </EuiFormRow>
+            {isDefined(fieldConfig.splitField) && selectedPartitions.length === 0 ? (
+              <EuiFormRow
+                fullWidth
+                label={
+                  <FormattedMessage
+                    id="xpack.aiops.changePointDetection.maxSeriesToPlotLabel"
+                    defaultMessage="Max series"
+                  />
+                }
+              >
+                <EuiFieldNumber
```

Comment:
Would make sense to set a maximum number of series that can be added. Would 50 be a sensible limit - consistent with that for anomaly charts?

Reply:
Added in https://github.com/elastic/kibana/pull/162796/commits/7605deeba8aebaaa994c3982a512349a39edaa67

Reply:
The number input now correctly shows a warning if the number is `> 50` or `< 1` but the 'Attach' button stays enabled on invalid input and allows you to proceed to the 'Select dashboard' popover. Can the button be disabled on invalid input?

<img width="229" alt="image" src="https://github.com/elastic/kibana/assets/7405507/2220b6f2-8a05-44ac-a9bf-b70e7cc1cad4">

Reply:
Updated in https://github.com/elastic/kibana/pull/162796/commits/002e192886d32bbad5cb9b760b5acb62413d316c